huxley/www/templates/www.html

```diff
@@ -19,7 +19,7 @@
             ContactTypes = {{ contact_types|safe }};
             ProgramTypes = {{ program_types|safe }};
         </script>
-        {% javascript 'huxley' %}
+        <script type="text/javascript" src="/static/js/bundle.js"></script>
```

Comment:
If this is an absolute path what is the root directory here? `www`? Just want to make sure this path points to `./huxley/www/static/js` as it does [here] (https://github.com/bmun/huxley/commit/2da93a3b950e9c53ab79bb60e6312cff5ce58609#diff-11e9f7f953edc64ba14b0cc350ae7b9dR22)